src/webui/src/components/overview/config/TrialConfigButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useCallback } from 'react';
+import { DefaultButton, Stack } from '@fluentui/react';
+import TrialConfigPanel from './TrialConfigPanel';
+import '../../../static/style/overview/config.scss';
+
+export const TrialConfigButton = (): any => {
+    const [isShowConfigPanel, setShowConfigPanle] = useState(false);
```

Comment:
Why using hook here?

Reply:
hooks is useful for function component with [true, false] states. I upgrad react version is main to use hooks:)